providers/telnyx/src/lib/telnyx.provider.ts

```diff
@@ -92,9 +92,7 @@ export class TelnyxSmsProvider implements ISmsProvider {
       case 'delivered':
         return SmsEventStatusEnum.DELIVERED;
       case 'delivery_unconfirmed':
-        return SmsEventStatusEnum.DELIVERY_UNCONFIRMED;
-      case 'received':
```

Comment:
SMS providers don't have a received confirmation?

Reply:
Because we are not interested in knowing that some one sent an sms to our Novu user :)